doc/ExpertOptions.md

```diff
@@ -1441,24 +1441,50 @@ For an in depth discussion, please see [Missing Call Arguments](MissingCallArgum
 
 # Manifests
 
-In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. In such a case, you can enable manifest support at compile time with the `-useManifests` option. This option causes each generated applet or workflow to accept inputs as a manifest, and to produce outputs as a manifest.
-
-A manifest is a JSON document that contains all the inputs/outputs that would otherwise be passed directly to/from the applet. A manifest can be specified in one of two ways: via a JSON input, or via a File input (where the file must exist on the platform).
+In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and 
+output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. 
+In such a case, you can enable manifest support at compile time with the `-useManifests` option. 
+This option causes each generated applet or workflow to accept inputs as an array of manifests, and to produce outputs as a single manifest.
```

Comment:
```suggestion
In extreme cases, running compiled workflows can fail due to DNAnexus platform limits on the total size of the input and output JSON documents of a job. An example is a task with many inputs/outputs that is called in scatter over a large collection. In such a case, you can enable manifest support at compile time with the `-useManifests` option. This option causes each generated applet or workflow to accept inputs as an array of manifests, and to produce outputs as a single manifest.
```

It's a minor thing but it's easier to read (now and in the future) if there are no newlines introduced.

Reply:
Here and below wherever the breakline is added.

Reply:
Actually, this new line here looks ugly to show the difference in versions. In the raw source it looks fine and there's no change in the rendered variant. But without this break line - the source looks like one stretched line and you have to scroll sidewise to read it. So I would really insist on having those break lines in the markdown docs from now on.

Reply:
This is how it looks now: https://github.com/dnanexus/dxCompiler/blob/455de1d4d71bebc2261d2702d2a6ea3e477a1c4c/doc/ExpertOptions.md?plain=1#L1444

This is how the rest of the doc looks without new lines:
https://github.com/dnanexus/dxCompiler/blob/455de1d4d71bebc2261d2702d2a6ea3e477a1c4c/doc/ExpertOptions.md?plain=1#L1296

To me second example is harder to read compared to my version

Reply:
please click the links to actually see it in the raw .md